src/main/resources/schema.sql

```diff
@@ -1,13 +1,43 @@
-DROP TABLE IF EXISTS hmhb_user;
+DROP TABLE IF EXISTS hmhb_user; /* get rid of this in the next PR */
```

Comment:
I'm leaving this in for at least 1 PR so that it will clean up the hmhb_user table in heroku.  If you don't like that, I can clean it up with heroku pg:psql later.